Orange/widgets/data/owimpute.py

```diff
@@ -186,6 +191,37 @@ def __init__(self):
             button_group.addButton(button, method)
             box_layout.addWidget(button, i % 3, i // 3)
 
+        def set_to_fixed_value():
+            self.set_default_method(Method.Default)
+
+        def set_default_time(datetime):
+            self.default_time = datetime.toMSecsSinceEpoch()
+            if self.default_method_index != Method.Default:
+                set_to_fixed_value()
+            else:
+                self._invalidate()
+
+        hlayout = QHBoxLayout()
+        box.layout().addLayout(hlayout)
+        button = QRadioButton("Fixed values; numeric variables:")
+        button_group.addButton(button, Method.Default)
+        button.setChecked(Method.Default == self.default_method_index)
+        hlayout.addWidget(button)
+
+        le = gui.lineEdit(
+            None, self, "default_numeric", valueType=float,
```

Comment:
`self.default_numeric` is a string by default, but this changes it to a float. This is inconsistent and probably helped with the below bug on line 308 (and others?).

Reply:
I initially planned to have a float, but then decided for str and forgot to remove this argument.